source/documentationplan.rst

```diff
@@ -0,0 +1,67 @@
+Documentation plan
+==================
+
+Code documentation
+------------------
+
+We chose to have a not too strict form of documentation.
+Everyone is responsible for the documentation of his own code/functionality.
+Documenting the code of another person is discouraged.
+Also documentation that doesn’t add additional information is left behind (see the following example):
+
+.. code-block:: c
+
+  /// gets age of Person
+  int getAge() const;
+
+For reference documentation we chose to use Doxygen. So we also use the associated syntax to provide more information for the arguments etc. We try to use the currently present documentation style as much as possible.
```

Comment:
"For reference documentation we chose to use Doxygen. **This means** we also use the associated syntax to provide more information for the arguments etc. We try to use the currently present documentation style as much as possible."